net/grpc/gateway/docker/prereqs/Dockerfile

```diff
@@ -45,6 +45,8 @@ RUN git clone https://github.com/grpc/grpc-web .
 COPY ./scripts/init_submodules.sh ./scripts/
 RUN ./scripts/init_submodules.sh
 
+# Add this to ensure abseil-cpp is available
+RUN git clone --depth 1 https://github.com/abseil/abseil-cpp.git third_party/abseil-cpp
```

Comment:
Would it be possible to check this out in `net/grpc/gateway/docker/protoc_plugin/Dockerfile` so that other docker flows which doesn't require building protoc_plugin isn't slowed down by it? :)